### DISCUSSION TITLE
Organization Plans

### DISCUSSION THREAD

**Discussion by suditaprayoga** [opengisch/QField#3932] [Q&A]
Hello,

I'm planning to create an organization with twenty members. but for the beginning I plan to add 4 members first. I have a few questions :
1. When i add new member, will it charge automatically (including 40% discount) ?
2.  Is it possible if the member account that will be added to the organization, only use the community account?
3. Can I manage a team for each member of the organization?
Looking forward for hearing the answer from you soon, Thank you in advance.

Prayoga

**Comment by ZsanettMed** [comment]
Hi, 

1. It is important that you create the organization today to get the EarlyBee offer (40% discount/user). It does not matter if you add the other 16 member later, the offer will be valid for you the whole year. Short note, since you will be charged only for the active users (but min for 2) you can also add all of them now. If they are not active, they won´t be charged.
2. Yes it is, the members can be either community or pro user
3. I am not sure if I understand this question correctly, but "team management" is included in the organization plan. 

Best, 
Zsanett
